### ISSUE TITLE
[Feature] Improve relevance of search results on website

### ISSUE THREAD

**Issue by dtaivpp** [opensearch-project/project-website-search#43]
**Is your feature request related to a problem? Please describe.**
At the moment the search bar on the OpenSearch website is not always providing the most relevant information. 

**Describe the solution you'd like**
We should evaluate searches that are under performing and work on tuning the ingestion and search template to improve search. 

1. Identify under performing searches
2. Evaluate why they are under performing
3. Work on improving the search via ingestion strategies and search templates

**Describe alternatives you've considered**
We could leave it as is however the user experience would suffer. 

**Additional context**
n/a

**Comment by dtaivpp** [comment]
@dblock  can you assign me to this one?

**Comment by macohen** [comment]
@dtaivpp any recommendations or discoveries so far?

**Comment by dtaivpp** [comment]
Yes! Sorry for the delay, here are some of the under performing queries we've identified so far. 

No Results Queries: 
- hight
- analizer
- buld
- aggra
- dense_vector
- densevector
- encryption at rest
- number of rows
- n-gram
- results at a time
- vector

Several of these could be remedied with the addition of fuzzyness. Additionally we could probably get better relevance if we ingest date for blogs and boost more recent ones.

**Comment by hdhalter** [comment]
Several community members have asked about searching only the documentation: https://forum.opensearch.org/t/new-documentation-home-page-design-request-for-feedback/17420/8

**Comment by dtaivpp** [comment]
> Several community members have asked about searching only the documentation: https://forum.opensearch.org/t/new-documentation-home-page-design-request-for-feedback/17420/8

Ill start looking into this. I feel like we should be able to add a filter or at least be able to check referring URL and boost documentation links.

@macohen just to double check there is no magic happening behind the scenes right now? I didn't see any mention of a search template anywhere.
